adadamp/_dist.py

```diff
@@ -311,3 +313,100 @@ def _get_gradients(
         grads = [grad_fn(idx=idx) for idx in worker_idxs]
         out = dask.compute(*grads)
         return out
+
+class DaskClassifier(DaskBaseDamper):
+    """
+    Dask based classifier built ontop of DaskBaseDamper. Changes include:
+    - Batch level statistics 
+    - Epoch level statistics
+    - Accuracy based scoring function
+    """
+    
+    def initialize(self, *args, **kwargs):
+        """
+        Initializes class instance variables
+        """
+        ret = super().initialize(*args, **kwargs)
+        
+        # batch level stats
+        self.stats_ = []
+        # epoch level stats
+        self.meta_ = {"n_updates": 0, "n_data": 0, "score_calls": 0, "score_times": [], "fit_calls": 0}
+        
+        return ret
+    
+    def get_stats(self):
+        """
+        Returns the epoch and batch level stats
+        """
+        meta_stats = deepcopy(self.meta_)
+        batch_stats = deepcopy(self.stats_)
+        
+        return meta_stats, batch_stats
+    
+    def fit(self, data, target):
+        """
+        Runs 1 epoch on the given data and model
```

Comment:
`partial_fit` should be one epoch of the training data. `fit` should be `max_epochs` passes over the training data. That mirrors [SGDClassifier][1].

[1]:https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html